packages/venia-ui/lib/components/CartPage/ProductListing/EditModal/productForm.js

```diff
@@ -71,14 +84,28 @@ const ProductForm = props => {
 
     return (
         <Fragment>
-            <FormError
-                classes={{
-                    root: classes.errorContainer
-                }}
-                errors={Array.from(errors.values())}
-                scrollOnError={false}
-            />
-            <Form onSubmit={handleSubmit}>
+            <Dialog
+                confirmText={'Update'}
+                confirmTranslationId={'global.updateButton'}
```

Comment:
Was it intentional to change this to use the global? If not, can you change this to `productForm.submit`?

Reply:
This was my mistake. I changed it to productForm.submit.